StudyConnect.API/Controllers/Forum/PostController.cs

```diff
@@ -7,7 +7,7 @@ namespace StudyConnect.API.Controllers.Forum;
 /// Provides endpoints to create, retrieve, update, and delete posts.
 /// </summary>
 [ApiController]
-[Route("v1/forum")]
+[Route("api/v1/posts")]
```

Comment:
```suggestion
[Route("v1/posts")]
```
shouldn't need api/ here I believe.

Reply:
it is needed
the "api/" is defined in Base- Controller, but it is overwritten cause i add Route for the whole class.
when i remove the "api" part, swagger looks like that:
![image](https://github.com/user-attachments/assets/bc9f3ac8-c7d1-409c-99c7-9092caa73cba)